server/src/db/migrations/20230915092918-remove_person_organisation.ts

```diff
@@ -0,0 +1,13 @@
+import { Db, MongoClient } from "mongodb";
+
+export const up = async (db: Db, _client: MongoClient) => {
+  await db.collection("persons").updateMany(
+    {},
```

Comment:
Tu peux speed-up au cas ou avec:
```suggestion
    { organisation: { $exists: true } },
```